Python/pywarpx/_libwarpx.py

```diff
@@ -1181,7 +1181,7 @@ def get_gathered_charge_density_fp(level):
 
     '''
 
-    return _get_mesh_field_list(libwarpx.warpx_getGatheredChargeDensityFP, level, None, True)
+    return _get_mesh_field_list(libwarpx.warpx_getGatheredChargeDensityFP, level, None, False)
```

Comment:
This will break the direct solver. The solution used for `get_gathered_phi_fp` is definitely a better way to go.